### ISSUE TITLE
Create Congo Same-Different Task with Short Musical Stimuli 

### ISSUE THREAD

**Issue by davidjohnbaker1** [Amsterdam-Music-Lab/MUSCLE#626]
## TL;DR

Request for a new rule set for an experiment in which a user has to judge whether presented audio stimuli are same or different using five buttons.

## Context

[Re-phrasing comment on older issue by @hhoning ]

We would like to create a micro-PROMS inspired experiment Congo field trip in Spring. 

The experiment will be a series of same/different judgments with music that is familiar to the BaYaka. 

## Paradigm / Trial flow: 

[Structure: Sound A, Sound A repeated, followed by Sound B. Judgement: Same or different? Sure, Doubt, or Don't know.] 

1. One audio file gets played, that contains:
  a. Sound A,
  b. Sound A repeated,
  c. followed by Sound B
2. Participant judges the sameness/differentness of Sound A and Sound B using the following buttons: `Definitely same`, `Probably same`, `Probably different`, `Definitely different`, and `I don’t know`.

## Good to know

If possible, we would like to run a public data collection on this on the AML server BEFORE going the Congo trip in Spring (we will have the stimuli by the end of the calendar year.  MA student William Wells is currently designing/piloting it. 

N.B. If this experiment would be a matter of just cloning and modifying something from the Rhythm perception battery / Goldsmiths MDT, please let me know because this would be a great project for myself and @Lia-JX-Li get involved with actually working with the MUSCLE framework.

**Comment by BeritJanssen** [comment]
There is a same/different experiment in the code base, namely the [rhythm discrimination](https://github.com/Amsterdam-Music-Lab/MUSCLE/blob/develop/backend/experiment/rules/rhythm_discrimination.py) experiment. That one includes a training phase, however, and the phrasing is also something you'd probably want to change. So I think it comes down to creating a new rules file. This would be a great project for getting to know the ropes of the MUSCLE framework. If you or @Lia-JX-Li are up for setting this up yourselves, perhaps it's an idea to look through the rhythm discrimination rules file and let us know how we can help to go from there to your own same-different experiment?

**Comment by davidjohnbaker1** [comment]
Just wanted to update here that I don't think that I will have the time/resources to properly do this without delaying other parts of the project. Could @BeritJanssen change the assignee on this?

**Comment by hhoning** [comment]
Is it still be feasible to have this exp in place at the end of the month (as previously planned)? If not, what is our plan B @davidjohnbaker1 ?

**Comment by davidjohnbaker1** [comment]
If this is not feasible for @BeritJanssen or others (e.g. @drikusroor) to do properly, I can spend next week making a MVP version of this. It's do-able by me, but will just take me longer and result in a poorer quality project.

**Comment by BeritJanssen** [comment]
@davidjohnbaker1 we're happy to take over. Though I noticed during discussion with @hhoning that the specs are actually a bit hazy to me, in that I'm not sure what the player(s) of this experiment should look like. The `rhythm_discrimination` experiment I pointed you to as an example has *one* file per trial, in which we get to hear two repetitions of the reference rhythm, and then one presentation of the target rhythm. This audiofile is presented as an Autoplay, i.e., it starts playing at the start of each trial.

From last week's conversation it seemed to me as if you'd want two or even three play buttons instead. Personally, I'd think two play buttons (one reference, one target stimulus) would be clearest. Right now, however, we can only control if a play button can be played once or as many times as the participant likes, no in-between options. So if there's a hard requirement here that the reference stimulus needs to be repeated twice, we would either need to take the approach of "rhythm discrimination", or indeed have three players which can all be played once.

**Comment by davidjohnbaker1** [comment]
Let me double check with the team, then get back to you by the EOD!

**Comment by davidjohnbaker1** [comment]
Copying something from @hhoning on Teams, but we very much wasn to stay as close as possible to the Micro-PROMs structure, which is described:

cf. "participants are presented a reference stimulus twice, separated by an interstimulus interval of 1.5 sec, followed by the comparison stimulus after an interval of 2.5 sec. The reference stimulus was presented twice to facilitate its encoding, thereby leaving less room for individual differences in memory capacity to affect the performance." and "Participants are asked to indicate whether reference and comparison are the same or different by selecting one of five answer options: “Definitely same,” “Probably same,” “Probably different,” “Definitely different,” and “I don’t know.”"

So there should be one button to start the trial, this plays a single stimuli (we will prepare a single file) then they are given the five options.
